test/versioned/pg/pg-post-7/package.json

```diff
@@ -5,7 +5,18 @@
   "tests": [
     {
       "engines": {
-        "node": ">=8"
+        "node": ">=10"
+      },
+      "dependencies": {
+        "pg": ">=8"
```

Comment:
I see here' we've added a new 10+ configuration for 8+ versions of `pg`.

Given `pg-native` hasn't been published in a couple years, I'm guessing it does not yet have a `v14` compatible version.